joss_paper/paper.md

```diff
@@ -0,0 +1,131 @@
+However, making simultaneous spatial, temporal and spectral measurements with a single
+instrument is often challenging.
+One solution is to disperse the incoming light through a narrow slit onto a pixelated
+detector creating a spectrogram, i.e. an image with a spectral dimension and a spatial
+dimension corresponding to the long axis of the slit.
+The second spatial dimension is reconstructed by scanning the slit across a region of
+interest and taking sequential exposures.
+A complete scan, known as a raster, creates a space-space-spectral data cube, and a time
+dimension can be generated by repeating raster scans over the same region of interest.
+Rastering or Scanning slit-spectrographs, as such instruments are known, have been
+successfully employed for solar UV/EUV observations, e.g. the EUV Imaging
+Spectrograph onboard Hinode [EIS; @eis], the Interface Region Imaging Spectrograph
+[IRIS; @iris], and the SPectral Imaging for the Coronal Environment onboard Solar
+Orbiter [SPICE; @spice].
+
+However the rastering slit-spectrograph design leads to a fundamental complication,
+namely that one of the spatial dimensions is coupled with time.
+This can create ambiguity if the dynamic timescales of the observed phenomenon is
+less than or comparable to the rastering cadence.
+This requires scientists to think of their data as both 3-D (time/space-spectral-space)
+and 4-D (time-spectral-space-space/time) alternately, and sometimes simultaneously,
+depending on their science goals.
+
+By building on features inherited from the ndcube package, sunraster provides an API
+for seamlessly switching between the 3-D and 4-D paradigms without changing or
+hard-copying the underlying data.
+This saves users carrying around multiple copies of the same data which might become
+inconsistent during a data analysis workflow.
+A powerful slicing API enables users to extract regions of interest with single commands
+that not only alter the data array, but also the associated uncertainties, data mask
+and the coordinate transformations.
+This saves users from performing multiple tedious and error-prone tasks on the various
+compoennts of their dataset individually.
+This makes the development of analysis workflows quicker, more reliable, and easier
+to understand, thus boosting users' productivity.
+
+# Community Applications
+
+Most of sunrasters tools are instrument-agnostic.
+However it does provide specific tools for reading Solar Orbiter/SPICE FITS files
+and hence helping SPICE users leverage the wider scientific Python ecosystem
+for their analysis.
+In addition, the Python user tools package for the IRIS mission, irispy-lmsal
+[irispy-docs; @irispy-code], depends on sunraster.
+
+# Acknowledgements
+
+We acknowledge financial support for sunraster from Solar Orbiter/SPICE
+(grant 80NSSC19K1000).
+We also acknowledge the SunPy and Python in Heliophysics communities for their support.
+
```

Comment:
```suggestion
We acknowledge financial support for sunraster from NASA's SDO/AIA and IRIS missions
(grant NNG04EA00C and NNG09FA40C).
```

Reply:
Moving this to be part of first sentence.